decompiler/analysis/insert_lets.cpp

```diff
@@ -2543,6 +3011,27 @@ LetStats insert_lets(const Function& func,
     });
   }
 
+  // Part 10: rewrite let sequences
+  top_level_form->apply_form([&](Form* f) {
```

Comment:
At first this seems kinda weird, but I don't know where else it could go. We want to do this after `let`s have been found, so it has to go here.